pkg/util/validation/limits.go

```diff
@@ -124,7 +124,7 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&l.EnforceMetricName, "validation.enforce-metric-name", true, "Enforce every sample has a metric name.")
 	f.BoolVar(&l.EnforceMetadataMetricName, "validation.enforce-metadata-metric-name", true, "Enforce every metadata has a metric name.")
 
-	f.IntVar(&l.MaxSeriesPerQuery, "ingester.max-series-per-query", 100000, "The maximum number of series for which a query can fetch samples from each ingester. This limit is enforced only in the ingesters (when querying samples not flushed to the storage yet) and it's a per-instance limit. This limit is ignored when running the Cortex blocks storage.")
+	f.IntVar(&l.MaxSeriesPerQuery, "ingester.max-series-per-query", 100000, "The maximum number of series for which a query can fetch samples from each ingester. This limit is enforced in the ingesters and it's a per-instance limit.")
```

Comment:
It's not just from ingesters. For blocks storage is from storage too. So we should update the doc as follow:
- When running Cortex chunks storage: limit enforced when fetching metrics from ingesters only
- When running Cortex blocks storage: limit enforced when fetching metrics both from ingesters and store-gateways

Each time you change a CLI flag you need to run `make doc` to auto-generate the updated doc.